modules/prediction/predictor/sequence/BUILD

```diff
@@ -22,4 +22,26 @@ cc_library(
     ],
 )
 
+cc_test(
+    name = "sequence_predictor_test",
+    size = "small",
+    srcs = [
+        "sequence_predictor_test.cc",
+    ],
+    data = [
+        "//modules/prediction:prediction_data",
+        "//modules/prediction:prediction_testdata",
+    ],
+    deps = [
+        "//modules/common/util",
+        "//modules/prediction/common:kml_map_based_test",
+        "//modules/prediction/common:prediction_gflags",
+        "//modules/prediction/container/obstacles:obstacles_container",
```

Comment:
correct the format here please

Reply:
@Capri2014  What kind of format did you refer to?